week-1/L-functions-nested/exercise.js

```diff
@@ -0,0 +1,26 @@
+/*function getAgeInDays(age) {
+    return age * 365;
+  }
+  
+  function createGreeting(name, age) {
+    var ageInDays = getAgeInDays(age);
+    var message =
+      "My Name is " + name + " and I was born over " + ageInDays + " days ago!";
+    return message;
+    
+  }
+  var nameAndAge =createGreeting("Saeed",31);
+  console.log(nameAndAge);  */
```

Comment:
Try to avoid committing lots of commented out code that isn't related to the task in hand.